packages/expo-modules-core/ExpoModulesCore.podspec

```
@@ -11,10 +17,18 @@ end
 
 reactNativeTargetVersion = reactNativeVersion.split('.')[1].to_i
 
+use_hermes = ENV['USE_HERMES'] == nil || ENV['USE_HERMES'] == '1'
 fabric_enabled = ENV['RCT_NEW_ARCH_ENABLED'] == '1'
-fabric_compiler_flags = '-DRN_FABRIC_ENABLED -DRCT_NEW_ARCH_ENABLED'
-folly_version = '2022.05.16.00'
+fabric_compiler_flags = '-DRCT_NEW_ARCH_ENABLED'
```

```suggestion
new_arch_compiler_flags = '-DRCT_NEW_ARCH_ENABLED'
```

also change fabric_enabled to `new_arch_enabled`